### ISSUE TITLE
Problem Details should be expanded

### ISSUE THREAD

**Issue by dyladan** [Dynatrace/davis-server#146]
The problem details intent doesn't really give you much useful information that the problem intent doesn't. Once we have "permission" from the user to do more analysis on a problem, perhaps we should spend more time doing things like looking for similar incidents in the past or listing more than just the first root cause event

**Comment by dyladan** [comment]
```
{
    "text": "Below are the problem details you requested.",
    "attachments": [
        {
            "fallback": "WEB_CHECK_GLOBAL_OUTAGE\nImpacted: easy travel demo booking \nApplication\nInitial Event: Web Check Global Outage\nStatus: Resolved",
            "color": "#dc172a",
            "title": "Web Check Global Outage",
            "title_link": "https://cdojfgmpzd.live.dynatrace.com/#problems;filter=watched/problemdetails;pid=-3951031557428477198",
            "fields": [
                {
                    "title": "Impacted",
                    "value": "easy travel demo booking \nApplication",
                    "short": "true"
                },
                {
                    "title": "Initial Event",
                    "value": "Web Check Global Outage",
                    "short": "true"
                },
                {
                    "title": "Status",
                    "value": "Unresolved",
                    "short": "true"
                },
                {
                    "title": "Unresolved Impacts",
                    "value": "1/5",
                    "short": true
                },
				{
					"title": "Time Frame",
					"value": "Started <!date^1484031471^{date_short_pretty} {time}|yesterday at 1:57 AM>. Currently ongoing",
					"short": false
				}
            ]
        },
        {
            "title": "User Action Duration Degradation",
            "color": "#7dc540",
            "fields": [
                {
                    "title": "Open",
                    "value": "0/4",
                    "short": true
                },
                {
                    "title": "Affected Actions/Minute",
                    "value": "25.4",
                    "short": true
                }
            ]
        },
        {
            "title": "Connection Lost",
            "color": "#dc172a",
            "fields": [
                {
                    "title": "Open",
                    "value": "1/1",
                    "short": true
                }
            ]
        }
    ]
}
```

[Check it out here](https://api.slack.com/docs/messages/builder?msg=%7B%22text%22%3A%22Below%20are%20the%20problem%20details%20you%20requested.%22%2C%22attachments%22%3A%5B%7B%22fallback%22%3A%22WEB_CHECK_GLOBAL_OUTAGE%5CnImpacted%3A%20easy%20travel%20demo%20booking%20%5CnApplication%5CnInitial%20Event%3A%20Web%20Check%20Global%20Outage%5CnStatus%3A%20Resolved%22%2C%22color%22%3A%22%23dc172a%22%2C%22title%22%3A%22Web%20Check%20Global%20Outage%22%2C%22title_link%22%3A%22https%3A%2F%2Fcdojfgmpzd.live.dynatrace.com%2F%23problems%3Bfilter%3Dwatched%2Fproblemdetails%3Bpid%3D-3951031557428477198%22%2C%22fields%22%3A%5B%7B%22title%22%3A%22Impacted%22%2C%22value%22%3A%22easy%20travel%20demo%20booking%20%5CnApplication%22%2C%22short%22%3A%22true%22%7D%2C%7B%22title%22%3A%22Initial%20Event%22%2C%22value%22%3A%22Web%20Check%20Global%20Outage%22%2C%22short%22%3A%22true%22%7D%2C%7B%22title%22%3A%22Status%22%2C%22value%22%3A%22Unresolved%22%2C%22short%22%3A%22true%22%7D%2C%7B%22title%22%3A%22Unresolved%20Impacts%22%2C%22value%22%3A%221%2F5%22%2C%22short%22%3Atrue%7D%2C%7B%22title%22%3A%22Time%20Frame%22%2C%22value%22%3A%22Started%20%3C!date%5E1484031471%5E%7Bdate_short_pretty%7D%20%7Btime%7D%7Cyesterday%20at%201%3A57%20AM%3E.%20Currently%20ongoing%22%2C%22short%22%3Afalse%7D%5D%7D%2C%7B%22title%22%3A%22User%20Action%20Duration%20Degradation%22%2C%22color%22%3A%22%237dc540%22%2C%22fields%22%3A%5B%7B%22title%22%3A%22Open%22%2C%22value%22%3A%220%2F4%22%2C%22short%22%3Atrue%7D%2C%7B%22title%22%3A%22Affected%20Actions%2FMinute%22%2C%22value%22%3A%2225.4%22%2C%22short%22%3Atrue%7D%5D%7D%2C%7B%22title%22%3A%22Connection%20Lost%22%2C%22color%22%3A%22%23dc172a%22%2C%22fields%22%3A%5B%7B%22title%22%3A%22Open%22%2C%22value%22%3A%221%2F1%22%2C%22short%22%3Atrue%7D%5D%7D%5D%7D)

**Comment by dyladan** [comment]
This is fixed for visual responses in 609e6159. Still need to expand text and audible.
